ph5/clients/ph5toms.py

```diff
@@ -643,17 +661,25 @@ def create_cut_list(self):
                                                        start_times, station_list, deployment, st_num))
                         elif self.reqtype == "FDSN":
                             # fdsn request
-                            cuts_generator.append(self.create_cut(seed_network, ph5_station, seed_station,
-                                                   start_times, station_list, deployment, st_num))
-                            
-        return itertools.chain.from_iterable(cuts_generator)
+                            stations_list = self.create_cut(seed_network, ph5_station, seed_station,
+                                                   start_times, station_list, deployment, st_num)
+                            for station in stations_list:
+                                cuts_generator.append(station)
+
+        del self.ph5.Experiment_t
+        del self.ph5.Array_t
+        del self.ph5.Event_t
+        self.ph5.clear()
+        self.clear()

```

Comment:
Is it necessary to explicitly force the deletion of these? I seems like it should get handled automatically.